src/components/BulkEditPane/BulkEditListResult/BulkEditMark/helpers.js

```diff
@@ -117,9 +117,21 @@ export const getNextDataControls = (action) => {
   }
 };
 
+export const isMarcValueValid = (value) => {
```

Comment:
typo in name, should be `isMarkValueValid`

Reply:
done!